src/CitMovie.Data/DataContext.cs

```diff
@@ -28,5 +28,16 @@ public class DataContext : DbContext
     
     public DataContext(DbContextOptions<DataContext> options) 
         : base(options) { }
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder) {
+        if (!optionsBuilder.IsConfigured)
+            optionsBuilder.UseNpgsql("host=localhost;uid=postgres;pwd=postgres;port=5433;db=portfolio_database");
```

Comment:
I think this is ok for now, we can refactor it so that the secrets are stored in a .env file like in the framework. 

#42 See PR if curios 😄

Reply:
Oh yes I was thinking about it :D Wanted to ask if we are going to do env file.

Reply:
Ok so I changed it to use the environment. I checked 42. I did not do everything as some of the settings are already done in 42. :D But when they both get merged it should work.